### ISSUE TITLE
Generate different files for different prefix

### ISSUE THREAD

**Issue by proyb6** [tachyons-css/tachyons-generator#62]
Could allow us to generate different CSS files that won't need to deliver all vendor prefixes if I'm using Chrome, there is no need for IE, Webkit and Firefox prefix, this help reduce files size.

Perhaps, a simple browser detection in our Javascript should deliver the CSS with vendors specific and full CSS for unknown web browser agent.

**Comment by johnotander** [comment]
Having different builds for different vendor prefix sets is a great idea! I do think we should handle this on the configuration level though, so you can create multiple builds with different prefixing schemes (which the server can dynamically route to).

**Comment by proyb6** [comment]
Good idea and how fast does the web browsers detect user agent if we're looking at performance? I've not experiment it before.
